systems/libseff/Dockerfile

```
@@ -0,0 +1,36 @@
+FROM ubuntu:18.04
```

Since we're using clang-14

```suggestion
FROM ubuntu:22.04
```